AmplifyPlugins/Core/AWSPluginsCore/Model/Support/QueryPredicate+GraphQL.swift

```diff
@@ -11,6 +11,11 @@ import Amplify
 public typealias GraphQLFilter = [String: Any]
 
 protocol GraphQLFilterConvertible {
+    func graphQLFilter(_ modelSchema: ModelSchema?) -> GraphQLFilter
```

Comment:
this may be my fault as I may have said to pass in optional ModelSchema, but by passing in `nil` means that it will call the deprecated computed property version of `GraphQLFilter` and expose the bug that you are fixing. Deprecrating the computed property is correct because there's a bug if it is used. Introducing a new public method that allows the bug to happen is incorrect then, so the signature should always require the model schema: `graphQLFilter(_ modelSchema: ModelSchema)`